hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/ListKeysResponse.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.recon.api.types;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * HTTP Response wrapped for listKeys requests.
+ */
+public class ListKeysResponse {
+  /** Path status. */
+  @JsonProperty("status")
+  private ResponseStatus status;
+
+  /** The current path request. */
+  @JsonProperty("path")
+  private String path;
+
+  /** Amount of data mapped to all keys and files in a cluster across all DNs. */
+  @JsonProperty("replicatedDataSize")
+  private long replicatedDataSize;
+
+  /** Amount of data mapped to all keys and files on a single DN. */
+  @JsonProperty("unReplicatedDataSize")
+  private long unReplicatedDataSize;
+
+  /** The number of keys based on limit under the request path. */
+  @JsonProperty("keyCount")
+  private int count;
+
+  /** last key sent. */
+  @JsonProperty("lastKey")
+  @JsonInclude(JsonInclude.Include.NON_EMPTY)
+  private String lastKey;
+
+  /** list of keys. */
+  @JsonProperty("keys")
+  private List<KeyEntityInfo> keys;
```

Comment:
> do count can be derived from keys field as its list and its size can give same info

It is being derived from key list size only, but need a separate field in order to avoid any computation at client UI side.

Reply:
removed `count` as discussed.